2021.1

siflfran opened the pull request.

omarelmasry commented:
**Some minor findings/comments:**

**LG 1-3:: 1st bullet:**
_"... impact factors are often related to dependability and safety
requirements, performance and real-time requirements, flexibility requirements and support for
different product variants and product lines."_
-> Suggestion to add "memory requirements" as well.

**LG 2-4:**
The word "options" is ambiguous. What kind of options are meant here?

**4.1:** 
Typo: extra slash  "/"

**LG 4-8:: last paragraph**
Shared resource do not affect WCET. It rather affects WCRT (worst-case response time). but depends on how WCET is defined in the material.

**LG 4-9**
extra hyphen in second paragraph -> "design and -analysis".

siflfran commented:
> **Some minor findings/comments:**
> 
> **LG 1-3:: 1st bullet:** _"... impact factors are often related to dependability and safety requirements, performance and real-time requirements, flexibility requirements and support for different product variants and product lines."_ -> Suggestion to add "memory requirements" as well.
> 
> **LG 2-4:** The word "options" is ambiguous. What kind of options are meant here?
> 
> **4.1:** Typo: extra slash "/"
> 
> **LG 4-8:: last paragraph** Shared resource do not affect WCET. It rather affects WCRT (worst-case response time). but depends on how WCET is defined in the material.
> 
> **LG 4-9** extra hyphen in second paragraph -> "design and -analysis".

I think I've addressed all of your points, if not please let me know.
